cluster-autoscaler/FAQ.md

```diff
@@ -1150,6 +1167,54 @@ Assumption: We assume that the developer executing the below stages wants to syn
 1. Create a PR on `machine-controller-manager-provider` branch of `gardener/autoscaler`
 
 
+### How do I sync gardener autoscaler with an upstream autoscaler micro release?
+
+This is helpful in order to offer Gardener CA with latest patch version of upstream. 
+
+Assumption: We assume that the developer executing the below stages wants to synchronise with a certain minor release `1.x.y` of the cluster autoscaler
+
+#### Stage A: Identify CA Release branch and Record Release commit id
+1.  Go to: [K8S Autoscaler Releases](https://github.com/kubernetes/autoscaler/releases) and navigate to the release page of the specific `1.x.y` patch release. Example: [cluster-autoscaler-1.26.1 Release Page](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.26.1).
+2.  Record the commit id of the release mentioned in this release page. For example, commit id of 1.26.1 release is [f48095c20ad1b305a1392a3a031b0a7e31e1927a](https://github.com/kubernetes/autoscaler/commit/f48095c20ad1b305a1392a3a031b0a7e31e1927a). We will call this as `patchReleaseCommitId`.
+3. Identify the release branch from [K8S Autoscaler Branches](https://github.com/kubernetes/autoscaler/branches). For example patch branch for `1.26` is [cluster-autoscaler-release-1.26](https://github.com/kubernetes/autoscaler/tree/cluster-autoscaler-release-1.26). We will cal lthis as `releaseBranch`
+
+#### Stage B: Setup Local Repo Before Merge
+1.  Fork `github.com/gardener/autoscaler` into your github account named `user` such that you now have a forked repo `https://github.com/user/autoscaler` . If you alread have the fork then Sync the fork.
+2.  Check out `github.com/gardener/autoscaler` under `$GOPATH/src/k8s.io`
+3.  Change to checked-out `autoscaler` dir and set `upstream` and `user` (fork) targets
+    1.  `git remote add user https://github.com/user/autoscaler`
+    2.  `git remote add upstream https://github.com/kubernetes/autoscaler.git`
+4. Execute: `git fetch –all`
+5. Switch to the release branch:
+6.  Switch to the `releaseBranch`: `git switch cluster-autoscaler-release-1.x.0`. Ex: `git switch cluster-autoscaler-release-1.26`
```

Comment:
the documentation of `git switch` says
```
THIS COMMAND IS EXPERIMENTAL. THE BEHAVIOR MAY CHANGE.
```
So I think it would be better if we stick to `git checkout -b` like also stated in the steps to sync with minor release...